packages/agoric-cli/src/deploy.js

```diff
@@ -367,12 +371,11 @@ export { bootPlugin } from ${JSON.stringify(absPath)};
                 console.info(`Loading plugin ${JSON.stringify(pluginFile)}`);
                 return E.get(E(pluginManager).load(pluginName, pluginOpts))
                   .pluginRoot;
-              } catch (e) {
+              }).catch(e => {
```

Comment:
```suggestion
              })().catch(e => {
```
Uh oh. A hazard of the new approach. Surprised the type checker didn't flag it, since the left operand of the `.catch` is a function. Any idea why not?

Reply:
Typescript checks weren't enabled for this file.  It has other type problems that I haven't addressed yet, but are likely outside the scope of this PR.